designer/import_dialog.ui

```diff
@@ -40,6 +40,23 @@
       </widget>
      </item>
      <item row="4" column="0">
+      <widget class="QLabel" name="label">
+       <property name="text">
+        <string>&amp;Author</string>
```

Comment:
Now that a lot more fields have shown up on here, let's mark out the optional fields with `(optional)`.

```suggestion
        <string>&amp;Author (optional)</string>
```